File: admin/src/main/resources/static/js.service/employeeService.js
app.service("employeeService",function ($http) {
    this.loadAllEmployee=()=>{
        alert("到service了");
        return $http.get("/employee/getAllEmployees")
    }

    this.findPage=(page,rows)=>{
        return $http.get("/employee/getAllEmployees?page=page+"+"rows="+rows);
    }
})

File: admin/src/main/java/com/oppo/employee/service/EmployeeService.java
package com.oppo.employee.service;

import com.oppo.employee.commom.PageResult;
import com.oppo.employee.pojo.Employees;
import com.oppo.employee.pojo.EmployeesInfo;

import java.util.List;

public interface EmployeeService {
    //获得所有的员工
    List<EmployeesInfo> getAllEmployeesInfo();

    List<Employees> getAllEmployees();

    List<EmployeesInfo> getAllEmployeesInfoObject();

    PageResult getAllEmployees(int page, int rows);
}
